dev/storage.h

```diff
@@ -1027,11 +1028,19 @@ namespace sqlite_orm {
                 auto compositeKeyColumnNames = tImpl.table.composite_key_columns_names();
                 sqlite3_reset(stmt);
                 tImpl.table.for_each_column([&o, &index, &stmt, &tImpl, &compositeKeyColumnNames, db](auto& c) {
+                    using table_type = typename std::decay<decltype(tImpl.table)>::type;
```

Comment:
are you sure this check is required here? The point is this function cannot be called without calling `statement_serializator` before. `statement_serializator` has this check already. Or this code got to be moved somewhere at one place to skip code copypasting

Reply:
Okay, I'll find a suitable place and move this check there, without copypasting